src/main/java/com/ethanyidong/bunny/BunnySession.java

```diff
@@ -0,0 +1,43 @@
+package com.ethanyidong.bunny;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class BunnySession {
+    private final static String DIVIDER = "____________________________________________________________";
+    private final static String GLOBAL_INDENTATION = "\t";
+    private final static String MESSAGE_INDENTATION = " ";
+
+    private ArrayList<Task> taskList;
+
+    public BunnySession() {
+        this.taskList = new ArrayList<>();
+    }
+
+    public void printMessage(String message) {
+        this.printMessage(Arrays.asList(message.split("\n")));
+    }
```

Comment:
Seeing from the occasions this function is called, I think it might be good to use the method signature of `System.out.printf` instead, so that we can use format strings instead of a *lot* of string concatenation `+` signs.